docs/specs/devcontainerjson-reference.md

```diff
@@ -167,3 +167,9 @@ You can see the VS Code implementation of the dev container schema [here](https:
 ## Publishing vs forwarding ports
 
 Docker has the concept of "publishing" ports when the container is created. Published ports behave very much like ports you make available to your local network. If your application only accepts calls from `localhost`, it will reject connections from published ports just as your local machine would for network calls. Forwarded ports, on the other hand, actually look like `localhost` to the application.
+
+## remoteUser
+
+A dev container configuration will inherit the `remoteUser` property from the base image it uses.
+
+Using the [images](https://github.com/devcontainers/images) and [Templates](https://github.com/devcontainers/images) part of the spec as an example: `remoteUser` in these images is set to a custom value of `vscode` - you may view an example in the [C++ image](https://github.com/devcontainers/images/blob/main/src/cpp/.devcontainer/devcontainer.json#L26). The [C++ Template](https://github.com/devcontainers/images/blob/main/src/cpp/.devcontainer/devcontainer.json#L26) will then inherit the custom `vscode` user from [its base C++ image](https://github.com/devcontainers/templates/blob/main/src/cpp/.devcontainer/Dockerfile#L1).
```

Comment:
ℹ️ Not all images in devcontainers/images have `vscode` as remoteUser. `typescript-node` & `javascript-node` have `remoteUser:node` and `universal` has `remoteUser: codespace`.

Wondering if this statement might be a lil confusing? 🤔 
Maybe we  shouldn't generalize this example for all images?

Reply:
Ah yeah I think that's a great point. Updated to be a bit more general - let me know what you think.

Reply:
Nice, like the new wordings.